plash_cli/core.py

```diff
@@ -110,17 +110,20 @@ def login(
 def deploy(
     path:Path=Path('.'), # Path to project
     local:bool=False,    # Local dev
-    port:int=5002):      # Port for local dev
+    port:int=5002,       # Port for local dev
+    app_id:str=None):    # App ID that will be used as the subdomain in plash
+
     """🚀 Ship your app to production"""
     print('Initializing deployment...')
+    if app_id == '': print('App ID cannot be an empty string'); return
     validate_app(path)
     tarz, filecount = create_tar_archive(path)
 
     plash_app = Path(path) / '.plash'
-    if not plash_app.exists():
+    if app_id is None and not plash_app.exists():
         # Create the .plash file and write the app name
```

Comment:
nitpick, and not your changed code, but can we remove this comment as it literally describes the code below it?